native/src/main/java/io/ballerina/stdlib/java/jdbc/nativeimpl/ClientProcessor.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2023 WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.stdlib.java.jdbc.nativeimpl;
+
+import io.ballerina.runtime.api.creators.ValueCreator;
+import io.ballerina.runtime.api.values.BMap;
+import io.ballerina.runtime.api.values.BObject;
+import io.ballerina.runtime.api.values.BString;
+import io.ballerina.stdlib.java.jdbc.Constants;
+import io.ballerina.stdlib.sql.datasource.SQLDatasource;
+import io.ballerina.stdlib.sql.utils.ErrorGenerator;
+
+import java.util.Locale;
+import java.util.Properties;
+
+/**
+ * This class will include the native method implementation for the Redshift client.
+ *
+ * @since 0.1.0
+ */
+public class ClientProcessor {
+
+    public static Object createClient(BObject client, BMap<BString, Object> clientConfig,
+                                      BMap<BString, Object> globalPool) {
+        String url = clientConfig.getStringValue(Constants.ClientConfiguration.URL).getValue();
+        if (!isJdbcUrlValid(url)) {
+            return ErrorGenerator.getSQLApplicationError("Invalid JDBC URL: " + url);
+        }
+        BString userVal = clientConfig.getStringValue(Constants.ClientConfiguration.USER);
+        String user = userVal == null ? null : userVal.getValue();
```

Comment:
Can't we define a separate function to retrieve the values from the `clientConfig` while handling the null? The logic seems to be repetitive here.